docs/release_notes/ibexa_dxp_v4.4.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Ibexa DXP v4.4 adds the improvements to the Welcome Page, PIM and Fastly.
+---
+
+# Ibexa DXP v4.4
+
+**Version number**: v4.4
+
+**Release date**: January 27, 2023
+
+**Release type**: [Fast Track](https://support.ibexa.co/Public/service-life)
+
+## Notable changes
+
+### New welcome page
+
+You will be welcomed to the Ibexa Digital Experience Platform by a newly designed welcome page.
+
+![New Welcome Page](4.4_welcome_page.png)
+
+### All-new Ibexa Commerce packages [[% include 'snippets/commerce_badge.md' %]]
+
+This release deprecates all Commerce packages that you've known from previous releases 
+and brings a completely redesigned and reconstructed Commerce offering.
+
+As part of this effort, two all-new components have been created: Cart and Checkout, 
+that you can use to build your own e-commerce presence. 
+Another addition is the Storefront component that serves as a proof-of-concept 
+and a basis for your customization and extension.
+
+![The new cart view](img/4.4_new_cart.png "The new cart view")
+
+For more information, see [Commerce](https://doc.ibexa.co/en/4.4/commerce/commerce/).
+
+### Fastly Image Optimizer (Fastly IO)
+
+You can now use Fastly IO to serve optimized versions of your images in real time and cache them.
+Fastly can perform multiple transformations on your image,
+for example, cropping, resizing and trimming before serving it to end user.
+Fastly is an external service that requires a separate subscription,
+to learn more see, [Fastly Image Optimizer website](https://docs.fastly.com/en/guides/about-fastly-image-optimizer).
+
+If you already have Fastly IO subscription, you can move to Fastly IO configuration in Ibexa DXP.
+
+#### Fastly VCL upload
+
+With this release, you can manipulate your Fastly VCL configuration directly from the command line.
+For example, you can define formats or source path for images.
+
+### New page blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+This release introduces new page blocks that rely on Personalization and PIM features 
+to let editors visually organize products on a page: 
+
+- [Catalog block](https://doc.ibexa.co/projects/userguide/en/4.4/content_management/block_reference/#catalog-block) displays products from a specific catalog to a selected customer group.
+- [Last purchased](https://doc.ibexa.co/projects/userguide/en/4.4/content_management/block_reference/#last-purchased-block) displays a list of products that were recently purchased, either generally, or by a specific user.
+- [Last viewed](https://doc.ibexa.co/projects/userguide/en/4.4/content_management/block_reference/#last-viewed-block) displays a list of products that were recently viewed.
+- [Product collection](https://doc.ibexa.co/projects/userguide/en/4.4/content_management/block_reference/#product-collection-block) displays a collection of specifically selected products.
```

Comment:
We forgot about one Product block: "Recently added" (based on Perso)

Reply:
When it comes to Page builder maybe we should also mention about new segmentation view (visibility tab) https://github.com/ibexa/segmentation/pull/69

Reply:
"Recently added" is described in API improvements https://github.com/ezsystems/developer-documentation/pull/1859/files#diff-c73e5a40002dde6ec8dc2d7eeacc7b9c345e0620c19f5da476d6f3262f004a9cR110

Reply:
@MagdalenaZuba but this is a new block available on UI (1 of 5):
![image](https://user-images.githubusercontent.com/104639617/215710781-69b71f13-765f-4c6b-adf9-c2c0f1dbb982.png)
Please mention it here.

Reply:
Will do 👍